samples/snippets/pom.xml

```diff
@@ -42,8 +42,11 @@
       <groupId>com.google.cloud</groupId>
       <artifactId>google-cloud-document-ai</artifactId>
     </dependency>
+    <dependency>
+      <groupId>com.google.cloud</groupId>
+      <artifactId>google-cloud-storage</artifactId>
```

Comment:
Please move this dependency outside of the snippet tag unless we want it to appear in the install instructions for all users.